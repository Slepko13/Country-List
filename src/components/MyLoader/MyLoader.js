import React from "react";
import './MyLoader.scss';


const MyLoader = ({loader}) => {
    return (
        <div className="MyLoader">
          {loader || "  Here will be my loader...."}
        </div>
    )
}
export default MyLoader;
