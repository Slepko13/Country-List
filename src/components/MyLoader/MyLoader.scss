.MyLoader{
  @media screen and (max-width: 991px){
    height: 200px;
  }
  width: 100%;
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
  text-align: center;
  font-size: 36px;
  color:blue;
  direction: ltr;

}