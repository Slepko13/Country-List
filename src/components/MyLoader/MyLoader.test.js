import React from "react";
import MyLoader from "./MyLoader";

const setUp = (props) => shallow(<MyLoader {...props}/>)

describe("my MyLoader component", () => {

    it("should render MyLoader Component with props(snapshot)", () => {
        let component = shallow(<MyLoader message="Custom loader"/>);
        expect(component).toMatchSnapshot();
    });

    it("should render MyLoader Component without props(snapshot)", () => {
        let component = shallow(<MyLoader />);
        expect(component).toMatchSnapshot();
    });
});

