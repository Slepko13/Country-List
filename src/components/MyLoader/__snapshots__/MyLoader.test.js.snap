// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my MyLoader component should render MyLoader Component with props(snapshot) 1`] = `
<div
  className="MyLoader"
>
    Here will be my loader....
</div>
`;

exports[`my MyLoader component should render MyLoader Component without props(snapshot) 1`] = `
<div
  className="MyLoader"
>
    Here will be my loader....
</div>
`;
